test/testapps/BasicTestApp/CascadingValueTest/CascadingValueSupplier.cshtml

```diff
@@ -16,7 +16,7 @@
     </CascadingValue>
 </CascadingValue>
 
-<p><button id="increment-count" onclick=@counterState.IncrementCount>Increment</button></p>
+<p><button id="increment-count" onclick=@(() => counterState.IncrementCount())>Increment</button></p>
```

Comment:
Is this an intentional change or did we create another gotcha here?

Based on what I can think of there's now a semantic difference between a capturing lambda and a method-group-to-delegate conversion in this case.

Reply:
This is the "breaking change" mentioned in the PR description.

There's now a semantic difference between passing delegates (e.g., lambdas) on the component type versus passing delegates on other types. There's no difference between lambdas versus methodgroup-to-delegate on the same component type. Please let me know if you think I'm incorrect or if you disagree with this design.

Reply:
No wait, there is a problem. Hmm... investigating options.